core/src/main/java/org/apache/calcite/sql/fun/SqlLibrary.java

```diff
@@ -72,6 +72,8 @@ public enum SqlLibrary {
   /** A collection of operators that are in PostgreSQL but not in standard
    * SQL. */
   POSTGRESQL("p", "postgresql"),
+  /** A collection of operators that are in RedShift but not in standard SQL. */
+  REDSHIFT("r", "redshift"),
```

Comment:
This seems only tangentially related to this PR.
I wonder whether this deserves a separate issue and PR.

Reply:
@mihaibudiu I ran into a problem with this:
https://github.com/apache/calcite/blob/main/babel/src/test/java/org/apache/calcite/test/BabelQuidemTest.java#L110

Both Oracle and PostgreSQL have `GREATEST` and `LEAST` functions but they behave differently. I needed some way for RedShift to be able to choose only one implementation.

Reply:
This has been moved to another PR.